core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -22,6 +22,7 @@
 #include "refcount.h"
 #include "rights.h"
 #include "str.h"
+#include "assert.h"
```

Comment:
Seems no assert function is called? And no need to include "assert.h", just use `bh_assert` if needed.